apps/cli/src/daemon-rpc.ts

```diff
@@ -2,7 +2,12 @@ import superjson from "superjson";
 import { createTRPCProxyClient, httpBatchLink } from "@trpc/client";
 import { appRouter } from "@opentrader/trpc";
 
-const DAEMON_URL = "http://localhost:8000/api/trpc";
+import { getSettings } from "./utils/settings.js";
+
+const settings = getSettings();
+
+const { domain, port } = settings;
+const DAEMON_URL = `http://${domain}:${port}/api/trpc`;
```

Comment:
We have four CLI commands that can send commands to the daemon:

- `opentrader exchange-add` - Add an exchange account
- `opentrader exchange-update` - Update an exchange account
- `opentrader trade` - Start the bot
- `opentrader stop` - Stop the bot

To ensure we always have the actual `host` and `port` from `settings.json`, this part must be moved inside the `createDaemonRpcClient()` function body.

Additionally, a new DaemonRpcClient instance must be created every time a command is called. Simply move this line inside the respective command function:
https://github.com/Open-Trader/opentrader/blob/dev/apps/cli/src/api/exchanges/add.ts#L29
Do the same for other commands, above.